extensions/AzureCosmosDBMongoVCore/AzureCosmosDBMongoVCore.TestApplication/appsettings.json

```diff
@@ -0,0 +1,68 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "Default": "Trace",
+            "Microsoft.AspNetCore": "Trace"
+        }
+    },
+    "UseRealEmbeddingGenerator": false,
+    "KernelMemory": {
+        "Services": {
+            "AzureCosmosDBMongoVCore": {
```

Comment:
I believe this should be renamed to `AzureCosmosDBMongoDBVCore`

Reply:
Updated the name